vanilla-framework/vanilla-framework-react
# Add Code Numbered component

# Done
- Added CodeBlock component with optional "numbered" modifier
- Added story to Storybook
- Added snapshot tests

# QA
- Pull code
- Run `yarn clean`, `yarn build` and `yarn serve`
- Run `yarn lint` and ensure there are no linting errors
- Verify the CodeBlock and CodeBlock Numbered pattern in Storybook
- Run `yarn test` and ensure the CodeBlock tests pass

# Fixes
Fixes #36, fixes #37, fixes #38
